src/form/addresses/index.ts

```diff
@@ -144,7 +144,7 @@ export const defaultAddressConfiguration: IAddressConfiguration[] = [
             `(${detailsDontExist} || ${mothersDetailsDontExistOnOtherPage}) || ${hideIfMotherAddressNotAvailable[0].expression}`
           ),
           expressionToConditional(
-            `(${detailsDontExist} || ${mothersDetailsDontExistOnOtherPage}) || ${hideIfMotherAddressNotAvailable[0].expression} || !${primaryAddressSameAsOtherPrimaryAddress})`,
```

Comment:
It seems that during the merge from `countryconfig` to `farajaland` an extra condition was added. Should this be added to 1.5 release or is this specific for `farajaland`?

Reply:
@makelicious Could you please check if the same issue is in farajaland release-v1.5 branch? If so let's amend it there as well

Reply:
The change was farajaland specific I think. I had encountered merge conflicts when pulling in changes from countryconfig. The malformed bracket was most likely from that :sweat_smile:

Reply:
it seems to be fine on the release branch at least https://github.com/opencrvs/opencrvs-farajaland/pull/1000/files#diff-84362893885c6ba15e20d5582fa9468ed65c2b1c9e7a2bdbd9f3c14cd3f2f860R146-R148

Reply:
Thank you for helping out!